src/sync.js

```diff
@@ -45,15 +46,26 @@ async function processBatch(txs) {
         await addUser(timestamp, payload.new_account_name);
         break;
       case "comment":
-        await addPost(
-          timestamp,
-          payload.parent_author,
-          payload.parent_permlink,
-          payload.author,
-          payload.permlink,
-          payload.title,
-          payload.body
-        );
+        if (!payload.parent_author) {
+          await addPost(
+            timestamp,
+            payload.parent_permlink,
```

Comment:
Do we need parent_permlink for posts?

Reply:
`parent_permlink` for posts is also called `category` see https://api.steemjs.com/getContent?author=mazar&permlink=journey-continue-tunnel-beach-otago-new-zealand . It's the first tag and it's not changeable, do we need it? Probably not if we handle the others tags but meanwhile..

Reply:
We can ask the same question for the field "title" in table "comments", as far as i can see it's always empty. I'm sure it's possible to set a title for comment but i never see that any app used it.

Reply:
I recall that when you were testing comments on `siol` in steemjs it was making bunch of comments that included titles.

Reply:
Yes https://github.com/steemit/steem-js/blob/master/test/comment.test.js#L28 Most of comments with title may come from here :)

Reply:
But useless